doc/nrf/known_issues.rst

```diff
@@ -955,14 +956,14 @@ KRKNWK-6073: Potential delay during FOTA
 Matter
 ======
 
-.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1
+.. rst-class:: v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1
```

Comment:
@greg-fer regarding all three known-issues related to Android CHIPTool. Should we update the version number, if Android CHIPTool was deprecated and we do not recommend using it anymore?

Reply:
Added the following note:

  .. note::
      The support for the Android CHIP Tool is removed as of the |NCS| v2.3.0 for Matter in the |NCS|. Use CHIP Tool for Linux or macOS instead, as described in :ref:`ug_matter_gs_testing`.